pkgs/development/python-modules/torch/bin.nix

```diff
@@ -55,6 +67,13 @@ in buildPythonPackage {
   postFixup = let
     rpath = lib.makeLibraryPath [ stdenv.cc.cc.lib ];
   in ''
+    pushd $out/${python.sitePackages}/torch/lib
+      LIBNVRTC=`ls libnvrtc-* |grep -v libnvrtc-builtins`
+      if [ ! -z "$LIBNVRTC" ] ; then
+        ln -s "$LIBNVRTC" libnvrtc.so
```

Comment:
I have a feeling that using the nix-packaged libnvrtc (`"${cudaPackages.cuda_nvrtc}/lib/libnvrtc.so"`) should be less fragile. At the very list, we have control over it